examples/src/pages/Viewer.tsx

```diff
@@ -51,7 +50,7 @@ export function Viewer() {
     // Check in order to avoid double initialization of everything, especially dat.gui.
     // See https://reactjs.org/docs/strict-mode.html#detecting-unexpected-side-effects for why its called twice.
     if (!canvasWrapperRef.current) {
-      return () => {};
+      return;
```

Comment:
Can revert this if you want, but I see no reason to return a function here instead of just a simple `return`